recipe/meta.yaml

```diff
@@ -1,82 +1,200 @@
+        - cmake
+        - ninja-base
+        - pkg-config
+      files:
+        - cmake_test/
+      commands:
+        # windows-only (almost-)all-in-one DLL + import library
+        - if not exist %LIBRARY_BIN%\\abseil_dll.dll exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\\abseil_dll.lib exit 1           # [win]
+        # absence of test targets in regular abseil output
+        - if exist %LIBRARY_BIN%\\abseil_test_dll.dll exit 1          # [win]
+        - if exist %LIBRARY_LIB%\\abseil_test_dll.lib exit 1          # [win]
+
+        # absl_* libraries
+      {% for each_lib in absl_libs %}
+        # presence of shared libs
+        - test -f $PREFIX/lib/libabsl_{{ each_lib }}${SHLIB_EXT}      # [unix]
+        # absence of static libs
+        - test ! -f $PREFIX/lib/libabsl_{{ each_lib }}.a              # [unix]
+
+        # some static libs remain on windows even for shared builds, see above
+        {% if each_lib in absl_libs_always_static_on_win %}
+        - if not exist %LIBRARY_LIB%\\absl_{{ each_lib }}.lib exit 1  # [win]
+        {% else %}
+        - if exist %LIBRARY_LIB%\\absl_{{ each_lib }}.lib exit 1      # [win]
+        {% endif %}
+
+        # pkg-config (should point to abseil_dll on shared windows builds)
+        - pkg-config --print-errors --exact-version "{{ v_major }}" absl_{{ each_lib }}
+      {% endfor %}
+
+        # absence of test targets in regular abseil output
+      {% for each_lib in absl_test_libs %}
+        # absence of all libs
+        - test ! -f $PREFIX/lib/libabsl_{{ each_lib }}${SHLIB_EXT}    # [unix]
+        - test ! -f $PREFIX/lib/libabsl_{{ each_lib }}.a              # [unix]
+        - if exist %LIBRARY_LIB%\\absl_{{ each_lib }}.lib exit 1      # [win]
+      {% endfor %}
+
+        # pkg-config (abseil_dll)
+        - pkg-config --print-errors --exact-version "{{ v_major }}" abseil_dll  # [win]
+
+        # CMake integration
+        - cd cmake_test
+        - export CMAKE_ARGS="$CMAKE_ARGS -GNinja -DCMAKE_BUILD_TYPE=Release"  # [unix]
+        - set "CMAKE_ARGS=%CMAKE_ARGS% -GNinja -DCMAKE_BUILD_TYPE=Release"    # [win]
+        - cmake $CMAKE_ARGS  -DCMAKE_CXX_STANDARD={{ cxx_standard }} .        # [unix]
+        - cmake %CMAKE_ARGS% -DCMAKE_CXX_STANDARD={{ cxx_standard }} .        # [win]
+        - cmake --build .
+        - ./flags_example    # [unix]
+        - flags_example.exe  # [win]
+
+  # tests & test-helpers
+  - name: libabseil-tests
+    script: build-abseil.sh  # [unix]
+    script: build-abseil.bat  # [win]
+    build:
+      string: cxx{{ cxx_standard }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      always_include_files:
+        # Must overwrite cmake metadata from libabseil
+        - lib/cmake/absl/          # [unix]
+        - Library/lib/cmake/absl/  # [win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja-base
+      host:
+        - gtest 1.14
```

Comment:
I can't see where gtest is being used, or the test binaries are being used (beyond checking that they're present)?

Reply:
It's for abseil_test_dll, included in this pacakge.

Reply:
do the test binaries get run? or does it ship a test suite for use by devs

Reply:
libprotobuf does make use of abseil test binaries to built its own test binaries